drivers/include/ng_at86rf2xx.h

```diff
@@ -137,6 +150,7 @@ typedef struct {
     uint8_t seq_nr;                     /**< sequence number to use next */
     uint8_t frame_len;                  /**< length of the current TX frame */
     uint16_t pan;                       /**< currently used PAN ID */
+    uint8_t chan;                       /**< currently used channel */
```

Comment:
I think, we should keep it as I edited ``get_chan`` to get ``dev->chan`` instead of asking the transceiver so every device will need it.